spec-0005/index.md

```diff
@@ -50,10 +50,10 @@ workflow" add the following lines as an additional step:
 
 ```
 - name: Upload wheel
-  uses: scientific-python/upload-nightly-wheel
+  uses: scientific-python/upload-nightly-action@main
```

Comment:
Can we suggest using releases rather than `main`?  Or we are foreseeing a lot of changes that would mean constant changes in the version in downstream config painful?

Reply:
Hmm, sure I don't have strong opinions about this. We do have a 0.1.0 release right now, but that's only tested with ipython and networkx right now, should we wait for some other package to try this out?